relayer/.eslintrc.json

```diff
@@ -1,28 +0,0 @@
-{
```

Comment:
Hm, why not JSON or was it accidentally reverted with everything else?

Reply:
I was fixing some issue and was experimenting with diff format, I will try to make it json in the my current branch